### ISSUE TITLE
useFqdnFor doesn't work for dependencies like "android-driver"

### ISSUE THREAD

**Issue by adenisyuk** [jmfayard/buildSrcVersions#85]
At 
`KotlinPoetry.kt`:

```

fun List<Dependency>.checkModeAndNames(useFdqnByDefault: List<String>): List<Dependency> {
    groupBy { d -> d.name }
        .forEach { (name, list) ->
            for (d: Dependency in list) {
                d.mode = when {
                    d.name in useFdqnByDefault -> VersionMode.GROUP_MODULE
                    list.size >= 2 -> VersionMode.GROUP_MODULE
                    else -> VersionMode.MODULE
                }
                d.escapedName = PluginConfig.escapeVersionsKt(
                    when (d.mode) {
                        VersionMode.MODULE -> d.name
                        VersionMode.GROUP -> d.group
                        VersionMode.GROUP_MODULE -> "${d.group}_${d.name}"
                    }
                )
            }
        }
    return this
}
```

dependency names in `useFdqnByDefault` are escaped, while `d.name` is not.

**Comment by jmfayard** [comment]
You are right, I stumbled on this issue as well

**Comment by jmfayard** [comment]
Fixed, will be there in 0.6.2 once it's released
